presto-main/src/main/java/com/facebook/presto/operator/ChannelSet.java

```diff
@@ -58,18 +61,24 @@ public boolean containsNull()
         return containsNull;
     }
 
+    public boolean hasElements()
+    {
+        return elements > 0;
+    }
+
     public boolean contains(int position, Page page)
     {
         return hash.contains(position, page, hashChannels);
     }
 
     public static class ChannelSetBuilder
     {
-        private static final int[] HASH_CHANNELS = { 0 };
+        private static final int[] HASH_CHANNELS = {0};
 
         private final GroupByHash hash;
         private final OperatorContext operatorContext;
         private final Page nullBlockPage;
+        private final LongAdder elements = new LongAdder();
```

Comment:
The builder seems to be used in a single thread only, so let's use `long` so that nobody thinks otherwise.